emsdk.py

```diff
@@ -1732,6 +1732,11 @@ def each_path_exists(pathlist):
       return all(os.path.exists(p) for p in pathlist)
 
     content_exists = os.path.exists(self.installation_path()) and each_path_exists(activated_path) and (os.path.isfile(self.installation_path()) or num_files_in_directory(self.installation_path()) > 0)
+    errlog(self.installation_path())
+    errlog(os.path.exists(self.installation_path()))
+    for p in activated_path:
+      errlog(p)
+      errlog(os.path.exists(p))
```

Comment:
looks like `errlog` prints to stderr - that still seems excessive here?

Reply:
Sorry, I'm just debugging the windows failure.